extensions/install.php

```diff
@@ -78,7 +78,7 @@ private function enablePlugins($parent)
 	 */
 	protected function getManifest($parent)
 	{
-		if (null === $this->manifest)
```

Comment:
When doing strict comparisons is always better to use the known type before the comparison operator to save php type conversion. 

`null === $something` is always faster than `$something === null` because it knows the type before the comparison.

We are talking about < 1 millisecond for 1 million comparisons but I still prefer to always use the option with better performance.

Fast benchmark result:

### $test === nulll
`$test === null: 0.051213026046753 seconds`

### null === $test
`null === $test: 0.049391984939575 seconds`

Can you please change it back?